# Configuration parameters for data recording node.

output_directory: '/root/SpiderGo'

topics:
#cyglidar
  # - '/cyglidar_h2/camera/points'
  # - '/cyglidar_h2_R/camera_R/points'
  # - '/cyglidar_h2_L/camera_L/points'
  # - '/clicked_point'
  # - '/cyglidar_h2/camera_L/amplitude_image_raw'
  # - '/cyglidar_h2/camera_L/camera_info'
  # - '/cyglidar_h2/camera_L/dcs_image_raw'
  # - '/cyglidar_h2/camera_L/distance_image_raw'
  # - '/cyglidar_h2/camera_L/points'
  # - '/cyglidar_h2/camera_R/amplitude_image_raw'
  # - '/cyglidar_h2/camera_R/camera_info'
  # - '/cyglidar_h2/camera_R/dcs_image_raw'
  # - '/cyglidar_h2/camera_R/distance_image_raw'
  # - '/cyglidar_h2/camera_R/points'
  - '/ENCODER'
  - '/spg_lidar_L/camera/camera_info'
  - '/spg_lidar_L/camera/points'
  - '/spg_lidar_R/camera/camera_info'
  - '/spg_lidar_R/camera/points'
  - '/usb_cam_L/camera_info'
  - '/usb_cam_L/image_raw/compressed'
  - '/usb_cam_R/camera_info'
  - '/usb_cam_R/image_raw/compressed'
  # - '/usb_cam_L/image_raw'
  # - '/usb_cam_R/image_raw'
  - '/imu'
  - '/scan' 
  - '/tf_static'
  - '/tf'
  # - '/usb_cam_R/image_raw/compressed/parameter_descriptions'
  # - '/usb_cam_R/image_raw/compressed/parameter_updates'
  # - '/usb_cam_R/image_raw/compressedDepth'
  # - '/usb_cam_R/image_raw/compressedDepth/parameter_descriptions'
  # - '/usb_cam_R/image_raw/compressedDepth/parameter_updates'
  # - '/usb_cam_R/image_raw/theora'
  # - '/usb_cam_R/image_raw/theora/parameter_descriptions'
  # - '/usb_cam_R/image_raw/theora/parameter_updates'


# Realsense
  # - '/d400/align_to_color/parameter_descriptions'
  # - '/d400/align_to_color/parameter_updates'
  # - '/d400/aligned_depth_to_color/camera_info'
  # - '/d400/aligned_depth_to_color/image_raw'
  # - '/d400/aligned_depth_to_color/image_raw/compressed'
  # - '/d400/aligned_depth_to_color/image_raw/compressed/parameter_descriptions'
  # - '/d400/aligned_depth_to_color/image_raw/compressed/parameter_updates'
  # - '/d400/aligned_depth_to_color/image_raw/compressedDepth'
  # - '/d400/aligned_depth_to_color/image_raw/compressedDepth/parameter_descriptions'
  # - '/d400/aligned_depth_to_color/image_raw/compressedDepth/parameter_updates'
  # - '/d400/aligned_depth_to_color/image_raw/theora'
  # - '/d400/aligned_depth_to_color/image_raw/theora/parameter_descriptions'
  # - '/d400/aligned_depth_to_color/image_raw/theora/parameter_updates'
  # - '/d400/color/camera_info'
  # - '/d400/color/image_raw'
  # - '/d400/color/image_raw/compressed'
  # - '/d400/color/image_raw/compressed/parameter_descriptions'
  # - '/d400/color/image_raw/compressed/parameter_updates'
  # - '/d400/color/image_raw/compressedDepth'
  # - '/d400/color/image_raw/compressedDepth/parameter_descriptions'
  # - '/d400/color/image_raw/compressedDepth/parameter_updates'
  # - '/d400/color/image_raw/theora'
  # - '/d400/color/image_raw/theora/parameter_descriptions'
  # - '/d400/color/image_raw/theora/parameter_updates'
  # - '/d400/color/metadata'
  # - '/d400/depth/camera_info'
  # - '/d400/depth/color/points'
  # - '/d400/depth/image_rect_raw'
  # - '/d400/depth/image_rect_raw/compressed'
  # - '/d400/depth/image_rect_raw/compressed/parameter_descriptions'
  # - '/d400/depth/image_rect_raw/compressed/parameter_updates'
  # - '/d400/depth/image_rect_raw/compressedDepth'
  # - '/d400/depth/image_rect_raw/compressedDepth/parameter_descriptions'
  # - '/d400/depth/image_rect_raw/compressedDepth/parameter_updates'
  # - '/d400/depth/image_rect_raw/theora'
  # - '/d400/depth/image_rect_raw/theora/parameter_descriptions'
  # - '/d400/depth/image_rect_raw/theora/parameter_updates'
  # - '/d400/depth/metadata'
  # - '/d400/extrinsics/depth_to_color'
  # - '/d400/motion_module/parameter_descriptions'
  # - '/d400/motion_module/parameter_updates'
  # - '/d400/pointcloud/parameter_descriptions'
  # - '/d400/pointcloud/parameter_updates'
  # - '/d400/realsense2_camera_manager/bond'
  # - '/d400/rgb_camera/auto_exposure_roi/parameter_descriptions'
  # - '/d400/rgb_camera/auto_exposure_roi/parameter_updates'
  # - '/d400/rgb_camera/parameter_descriptions'
  # - '/d400/rgb_camera/parameter_updates'
  # - '/d400/stereo_module/auto_exposure_roi/parameter_descriptions'
  # - '/d400/stereo_module/auto_exposure_roi/parameter_updates'
  # - '/d400/stereo_module/parameter_descriptions'
  # - '/d400/stereo_module/parameter_updates'
  # - '/diagnostics'
  # - '/gps/fix'
  # - '/imu/data'
  # - '/rosout'
  # - '/rosout_agg'
  # - '/rtabmap/cloud_ground'
  # - '/rtabmap/cloud_map'
  # - '/rtabmap/cloud_obstacles'
  # - '/rtabmap/global_path'
  # - '/rtabmap/global_path_nodes'
  # - '/rtabmap/global_pose'
  # - '/rtabmap/goal'
  # - '/rtabmap/goal_node'
  # - '/rtabmap/goal_out'
  # - '/rtabmap/goal_reached'
  # - '/rtabmap/grid_map'
  # - '/rtabmap/grid_prob_map'
  # - '/rtabmap/info'
  # - '/rtabmap/initialpose'
  # - '/rtabmap/labels'
  # - '/rtabmap/landmarks'
  # - '/rtabmap/local_grid_empty'
  # - '/rtabmap/local_grid_ground'
  # - '/rtabmap/local_grid_obstacle'
  # - '/rtabmap/local_path'
  # - '/rtabmap/local_path_nodes'
  # - '/rtabmap/localization_pose'
  # - '/rtabmap/mapData'
  # - '/rtabmap/mapGraph'
  # - '/rtabmap/mapOdomCache'
  # - '/rtabmap/mapPath'
  # - '/rtabmap/octomap_binary'
  # - '/rtabmap/octomap_empty_space'
  # - '/rtabmap/octomap_full'
  # - '/rtabmap/octomap_global_frontier_space'
  # - '/rtabmap/octomap_grid'
  # - '/rtabmap/octomap_ground'
  # - '/rtabmap/octomap_obstacles'
  # - '/rtabmap/octomap_occupied_space'
  # - '/rtabmap/proj_map'
  # - '/rtabmap/republish_node_data'
  # - '/rtabmap/scan_map'
  # - '/t265/odom/metadata'
  # - '/t265/odom/sample'
  # - '/t265/realsense2_camera_manager/bond'
  # - '/t265/tracking_module/parameter_descriptions'
  # - '/t265/tracking_module/parameter_updates'
  # - '/tag_detections'
  # - '/tf'
  # - '/tf_static'
  # - '/user_data_async'